src/hud_elements.cpp

```diff
@@ -1566,8 +1679,8 @@ void HudElements::sort_elements(const std::pair<std::string, std::string>& optio
         {"winesync", {winesync}},
         {"present_mode", {present_mode}},
         {"network", {network}},
-        {"display_server", {_display_session}}
-
+        {"display_server", {_display_session}},
+        {"fex_stats", {fex_stats}},
```

Comment:
`fex_stats` should be added to `legacy_elements()` as well

Reply:
Good to know. Added it. Wasn't sure if it was necessary because of the name "legacy".